terraform/etl/49-lambda-streetscene-ingestion.tf

```diff
@@ -141,8 +141,8 @@ module "street-systems-api-ingestion" {
   lambda_role_arn                = aws_iam_role.streetscene_street_systems_ingestion[0].arn
   environment_variables          = {
     API_SECRET_NAME       = "/data-and-insight/streets_systems_api_key"
-    OUTPUT_S3_FOLDER      = "${module.landing_zone_data_source.bucket_arn}/streetscene/*"
-    TARGET_S3_BUCKET_NAME = "streetscene/traffic-counters/"
+    OUTPUT_S3_FOLDER      = "${module.raw_zone_data_source.bucket_arn}/streetscene/*"
+    TARGET_S3_BUCKET_NAME = "traffic-counters/street-systems"
```

Comment:
do you need a trailing '/' on this?

Reply:
Amended

Reply:
need to push? and do you need that '*' on the end of line 144?